tst/testrandom.g

```diff
@@ -1,35 +1,58 @@
-randomTest := function(collection, method, checkin...)
-    local test1, test2, test3, test4, test5, test6, localgen, checkmethod;
-    if Length(checkin) = 0 then
-        checkmethod := \in;
-    else
-        checkmethod := checkin[1];
-    fi;
+# Perform a variety of tests on Random Sources and functions which create
+# random objects.
+#
+# This function is used for a variety is different tests:
+#
+# Test that 'Random(C)' and 'Random(GlobalMersenneTwister, C)' produce 
+# the same answer.
+#
+# Test that 'Random(rs,C)' only uses 'rs', and no other source of random
+#
+# Test Random, PseudoRandom and RandomList
```

Comment:
Actually, ˋPseudoRandomˋ does not yet allow a random source, and in fact for some methods (e.g. for groups) it may not be possible to reset the state of a random source and then get the same set of pseudo random elements, because there is additional mixing state in the group